nss-ice-phantoms/BangazonAPI
# Employees Controller and tests

# Description
Completed EmployeeController and associated tests.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout meg-employees`
1. Run BangazonAPI
1. Verify that GET (all employees and a single employee) works, and that each employee has a Department object and Computer object attached to it. If you want to test out an Employee without a computer, add a new Employee that is not associated with a ComputerEmployee. `insert into Employee (FirstName, LastName, DepartmentId, IsSuperVisor) values ('John', 'Doe', 3, 1)` 
1. Verify that POST and PUT both work.
1. Run four tests in Visual Studio.

# Checklist:
